src/cli.ts

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 import "reflect-metadata";
+import * as yargs from "yargs";
```

Comment:
do yargs include typings now?

Reply:
@pleerock https://github.com/typeorm/typeorm/pull/3220/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R45 yes

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aad7b771590a78176ee5beed711da512f54d7e15/types/yargs/index.d.ts

Reply:
ah sorry didn't saw that